docs/en_US/TrainingService/SupportTrainingService.md

```diff
@@ -0,0 +1,36 @@
+# Support TrainingService
+
+TrainingService is a concept of service that used to maintain a training platform for running trial jobs. NNI supports [local](./LocalMode.md), [remote](./RemoteMachineMode.md), [pai](./PaiMode.md), [kubeflow](./KubeflowMode.md) and [frameworkcontroller](./FrameworkControllerMode.md) built-in training services.   
```

Comment:
NNI **TrainingService** provides the training platform for running NNI trial jobs.

Reply:
fixed